app/jobs/fastly_log_processor.rb

```diff
@@ -62,18 +55,4 @@ def download_counts(log_ticket)
     end
   end
   statsd_count :download_counts, 'fastly_log_processor.download_counts'
-
-  # Takes a hash of download counts and turns it into an array of arrays for
-  # Download.bulk_update. E.g.:
-  #   [
-  #     ['rails', 'rails-4.0.0', 25 ],
-  #     ['rails', 'rails-4.2.0', 50 ]
-  #   ]
-  def munge_for_bulk_update(download_counts)
-    download_counts.map do |path, count|
-      name = Version.rubygem_name_for(path)
-      # Skip downloads that don't have a version in redis
-      name ? [name, path, count] : nil
-    end.compact
-  end
```

Comment:
I could remove this, because we can only use the full_path the lookup the version, and dont need the name too.